src/Microsoft.DotNet.InternalAbstractions/Native/NativeMethods.Windows.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.InternalAbstractions.Native
+{
+    internal static partial class NativeMethods
+    {
+        public static class Windows
+        {
+            [StructLayout(LayoutKind.Sequential)]
+            internal struct RTL_OSVERSIONINFOEX
+            {
+                internal uint dwOSVersionInfoSize;
+                internal uint dwMajorVersion;
+                internal uint dwMinorVersion;
+                internal uint dwBuildNumber;
+                internal uint dwPlatformId;
+                [MarshalAs(UnmanagedType.ByValTStr, SizeConst = 128)]
+                internal string szCSDVersion;
+            }
+
+            // This call avoids the shimming Windows does to report old versions
+            [DllImport("ntdll")]
+            private static extern int RtlGetVersion(out RTL_OSVERSIONINFOEX lpVersionInformation);
```

Comment:
Per https://github.com/dotnet/cli/pull/2747#discussion_r61533348, there's an issue with the package in RC2 where it doesn't pull in the runtime dependencies for net45x. Since a fair bit of code is meant to run in desktop (e.g. Testing.Abstractions in xunit runner, DependencyContext in web applications targeting desktop, tools calling Cli.Utils) - I'd rather not change what currently works. We should replace it for RTM where the issue with the RuntimeInformation package has been addressed.